### DISCUSSION TITLE
Blocks with OLE content

### DISCUSSION THREAD

**Discussion by hoellersbuero** [haplokuon/netDxf#354] [Q&A] [answered]
Hi,
First of all, thank you for this great library.
My Application reads a dxf file, makes some modifications and writes it to a new file.
Problem: the file which is read at the beginning contains a block with OLE content, a small picture. My app creates some insertion points for this block. After writing the new file and reopening it in ACAD, the OLE block seems to be empty and the insertion points are gone. 

When I open the original file in ACAD, the block seems to be empty during drag for place, but after placement the content is visible as it should be. Does anybody know the reason of this behaviour or/and a solution 
regards,
michael

**Comment by haplokuon** [comment, marked as answer]
The list of supported entities is in the README.md, the information of not supported entities is lost. OLE entities are not among them.
